Dockerfile.rhel

```diff
@@ -1,23 +1,46 @@
 ARG BUILDER_IMAGE=brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.23
-ARG BASE_IMAGE=registry.access.redhat.com/ubi9/ubi-micro:latest
+ARG BASE_IMAGE=registry.redhat.io/rhel9-4-els/rhel-minimal:9.4
 
 # Build the manager binary
-FROM --platform=${BUILDPLATFORM} ${BUILDER_IMAGE} AS builder
+FROM ${BUILDER_IMAGE} AS builder
 
-ARG CGO_ENABLED=1
+ARG TARGETOS
 ARG TARGETARCH
+ARG TARGETPLATFORM
 
-WORKDIR /workspace
+ENV GOEXPERIMENT=strictfipsruntime
 
-# Copy the Go source code
+WORKDIR /workspace
 COPY . .
 
-# Build the manager binary
-RUN make build GO_BUILD_ENV="CGO_ENABLED=${CGO_ENABLED} GOOS=linux GOARCH=${TARGETARCH}"
+RUN CGO_ENABLED=1 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -tags strictfipsruntime -mod=vendor -a -o manager cmd/kueue/main.go
 
-FROM --platform=${BUILDPLATFORM} ${BASE_IMAGE}
+FROM --platform=$TARGETPLATFORM ${BASE_IMAGE}
 WORKDIR /
-COPY --from=builder /workspace/bin/manager .
+COPY --from=builder /workspace/manager .
+RUN mkdir /licenses
+COPY --from=builder /workspace/LICENSE /licenses/.
 USER 65532:65532
 
-ENTRYPOINT ["/manager"]
+LABEL com.redhat.component="Kueue"
+LABEL name="kueue"
+LABEL release="0.10.1"
```

Comment:
why is this hard-coded?

Reply:
I took the value that LWS used. I'm not sure on this one.

Reply:
I am dropping those fields for now.